src/lib/Components/Toast.svelte

```diff
@@ -3,7 +3,9 @@
 
 <script>
   import SvgIcon from '$lib/Components/SvgIcon.svelte';
+	import { onMount } from 'svelte';
 	import { bell, checkMarkFilledAlt, exclamationFilled, infoFilled, x, xFilled } from './iconPaths';
+  import { fade, fly } from "svelte/transition";
 
```

Comment:
I can't see this well in Github, but check indentations.

Also arrange imports in a waterfall. shorter lines above longer lines